package.json

```diff
@@ -13,8 +13,10 @@
     "test:e2e": "cypress open --e2e --browser electron"
   },
   "dependencies": {
+    "@ant-design/icons": "^5.0.1",
     "@next-auth/firebase-adapter": "^2.0.1",
     "@plaiceholder/next": "^2.5.0",
+    "antd": "^5.3.2",
```

Comment:
Don't add ant design, we already have DaisyUI. Just download the SVGs and export them manually (like in the sidebar icons file).

You can get SVGs from https://fontawesome.com for free (not all of them but some)